pkg/operator/controllers/storageaccounts/storageaccount_controller.go

```diff
@@ -76,8 +79,16 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 		return reconcile.Result{}, err
 	}
 
-	if !instance.Spec.OperatorFlags.GetSimpleBoolean(controllerEnabled) {
-		// controller is disabled
+	// If disabled or managed is missing, do nothing
+	var managed *bool
+	switch strings.ToUpper(instance.Spec.OperatorFlags.GetWithDefault(controllerManaged, "")) {
```

Comment:
we should treat the `managed` operator flag as a tri-state bool.  essentially if it is set to true or false, it represents those values.  Else it's `nil` and we don't do anything.